python/tests/test_extractor.py

```diff
@@ -0,0 +1,13 @@
+from energuide import extractor
+
+
+def test_extract(energuide_fixture: str):
+    with open(energuide_fixture, 'r') as file_in:
```

Comment:
Let's test this with an explicit in-code fixture of 1 item with the minimum set of fields, instead of referring to a test file. Test files are better for integration tests.